applications/nrf_desktop/doc/settings_load.rst

```diff
@@ -0,0 +1,29 @@
+.. _settings_load:
+
+Settings load module
+####################
+
+Use the Settings load module to trigger loading the data from non-volatile memory.
+
+Module Events
+*************
+
++----------------+-------------+-------------------+-----------------+------------------+
+| Source Module  | Input Event | This Module       | Output Event    | Sink Module      |
++================+=============+===================+=================+==================+
+|                |             | ``settings_load`` |                 |                  |
++----------------+-------------+-------------------+-----------------+------------------+
+
+Configuration
+*************
+
+The Settings load module is enabled for every nRF Desktop device with :ref:`zephyr:settings` enabled - when :option:`CONFIG_SETTINGS` is set.
+
+Zephyr assumes that the application is responsible for calling the :cpp:func:`settings_load` after having done all necessary initialization.
```

Comment:
```suggestion
Zephyr assumes that the application will call :cpp:func:`settings_load` after completing all necessary initialization.
```
Any reason we can provide for this behavior? This `responsible` feels a bit off here, suggesting some kind of imperative, obligation, or requirement.

Reply:
I will add more detailed description here.